test/test_verify.py

```diff
@@ -64,3 +64,43 @@ def test_verify_result_boolish():
     assert not VerificationFailure(reason="foo")
     assert not CertificateVerificationFailure(reason="foo", exception=ValueError("bar"))
     assert VerificationSuccess()
+
+
+@pytest.mark.online
+def test_verifier_issuer(signed_asset):
+    a_assets = signed_asset("a.txt")
+
+    verifier = Verifier.staging()
+    assert verifier.verify(
+        a_assets[0],
+        a_assets[1],
+        a_assets[2],
+        expected_cert_oidc_issuer="https://github.com/login/oauth",
+    )
+
+
+@pytest.mark.online
+def test_verifier_san_email(signed_asset):
+    a_assets = signed_asset("a.txt")
+
+    verifier = Verifier.staging()
+    assert verifier.verify(
+        a_assets[0],
+        a_assets[1],
+        a_assets[2],
+        expected_cert_email="william@yossarian.net",
```

Comment:
@woodruffw Do you mind if I check this in a unit test? I can generate another certificate with my Trail of Bits email if you'd prefer that I don't expose this.

Reply:
Nope, I don't mind. It's already checked in, anyways 🙂